core/src/test/java/org/elasticsearch/cluster/MinimumMasterNodesIT.java

```diff
@@ -62,7 +62,6 @@
 
 @ClusterScope(scope = Scope.TEST, numDataNodes = 0)
 @ESIntegTestCase.SuppressLocalMode
-@TestLogging("_root:DEBUG,cluster.service:TRACE,discovery.zen:TRACE")
```

Comment:
it's pretty usless if you have that on a test forever? can we add it back when we need it?